### DISCUSSION TITLE
Column name problem.

### DISCUSSION THREAD

**Discussion by pk4all** [adonisjs/core#1283] [Help] [answered]
I have created a table with some fields. I have generate a modal for it. But I have facing a problem with column name, its return me in snake_case. How to avoid this?
My Modal code is
 ```javascript

@column({ columnName: 'countryCode' })
  public countryCode: number

```
it's return "country_code" but my expectation is "countryCode".

**Comment by RomainLanz** [comment, marked as answer]
Hey @pk4all! 👋 

This is normal since we are following SQL convention (where your column should be `snake_case`).
If you want to bypass this, you can freely change the behavior of our serializer to not mutate the column name.

Inside the `config/database.ts` file, you need to overide the `getSerializeAsKey` method.

```ts
orm: {
  getSerializeAsKey(_, key) {
    return key;
  },
},
```

**Reply by thetutlage** [reply]
Also, if you will read the documentation carefully, you will find all the answers https://preview.adonisjs.com/guides/models/columns#column-options
https://preview.adonisjs.com/guides/models/serializing-models

**Reply by pk4all** [reply]
Thanks for your response.
Actually I have convert a project APIs from JAVA to node, it's database already defined. If I have change column name in "snake_case" we need to more work on our App and frontend. So I need non serialize column name.
